code/modules/mob/living/carbon/superior_animal/roach/types/nanites.dm

```diff
@@ -58,7 +73,17 @@
 	max_n2 = 0
 	minbodytemp = 0
 
+	var/mob/living/carbon/superior_animal/roach/nanite/parent
```

Comment:
Now tell me: where is this var set exactly?

Reply:
Whoops. Forgot to add it to their New()

Will sort out when back